src/components/Body/MovieCards/MovieCards.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import movies from "../../../assets/movies/movies";
+import { Component } from "react";
+import MovieCard from "./MovieCard/MovieCard";
+import { MovieCardsState, MovieCardsProps } from "./movieCards.types";
+import styles from "./movieCadrds.module.css";
+
+class MovieCards extends Component<MovieCardsProps, MovieCardsState> {
+  state: MovieCardsState = {
+    movies: [],
+  };
+
+  componentDidMount() {
+    this.setState({ movies });
+  }
+
+  render() {
+    return (
+      <div className={styles.movieCards}>
+        {this.state.movies.map((movie) => (
+          <MovieCard
+            key={movie.id}
+            imageUrl={movie.imageUrl}
+            name={movie.name}
+            year={movie.year}
+            genre={movie.genre}
```

Comment:
Why don't you pass all this data with one movie object?

```javascript
function MovieCards() {
  return (
    <div className={styles.movieCards}>
      {movies.map((movie) => (
        <MovieCard
          key={movie.id}
          movie={movie}
        />
      ))}
    </div>
  );
}
```
What do you think the component looks better now?